note/doc/index.md

```diff
@@ -0,0 +1,83 @@
+```js script
+import { html } from 'lit';
+import '@divriots/dockit-core/note/dockit-note.define';
+```
+
+# dockit-note
+
+Component used to call out information in documentation pages.
+
+## Setup
+
+```js
+import '@divriots/dockit-core/note/dockit-note';
+```
+
+## Demo
+
+```js preview-story
+export const note = () => html`
+  <dockit-note>
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit
+  </dockit-note>
+`;
+```
+
+### Variants
+
+#### Info
+
+```js preview-story
+export const note_info_variant = () => html`
+  <dockit-note variant="info">
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit
+  </dockit-note>
+`;
+```
+
+#### Warning
+
+```js preview-story
+export const note_warning_variant = () =>
+  html` <dockit-note variant="warning">
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit
+  </dockit-note>`;
+```
+
+#### Error
+
+```js preview-story
+export const note_error_variant = () =>
+  html` <dockit-note variant="error">
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit
+  </dockit-note>`;
+```
+
+### Note with custom color
+
+```js preview-story
+export const note_custom_color = () => html`
+  <dockit-note color="#36B37E40">
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit
+  </dockit-note>
+`;
+```
+
+### Note with custom style
+
+```js preview-story
+export const note_themed = () => html`
+  <style>
+    .themed-note dockit-note div {
```

Comment:
this does't look right for 2 reasons
- nitpick: why not add `themed-note` directly on dockit-note?
- where is this `div` coming from? if users puts html content in the tag, then rendering smth different into it by the component is a bad practice, users might have their own html template that will compete for this pat of the DOM with the component internals